workflow/react-project/src/store/randomCocktail/utils.jsx

```diff
@@ -0,0 +1 @@
+export const normalizeRandomCocktailData = data => data?.drinks?.[0] || {};
```

Comment:
extend function to return all necessary data, in forms you need (move from reducer to here)